src/smclarify/bias/metrics/common.py

```diff
@@ -151,28 +151,30 @@ def series_datatype(data: pd.Series, values: Optional[List[Any]] = None) -> Data
     return data_type
 
 
-def ensure_series_data_type(data: pd.Series, values: Optional[List[Any]] = None) -> Tuple[DataType, pd.Series]:
+def ensure_series_data_type(
+    series: pd.Series, sensitive_facet_values: Optional[List[Any]] = None
+) -> Tuple[DataType, pd.Series]:
     """
     Determine the type of the given data series using set of rules, and then do necessary type conversion
     to ensure the series data type.
-    :param data: data for facet/label/predicted_label columns
-    :param values: list of facet or label values provided by user
+    :param series: data for facet/label/predicted_label columns
+    :param sensitive_facet_values: list of facet or label values provided by user
```

Comment:
We use this function for both labels and facets. So, it could be sensitive_facet_values or positive_label_value. This is why it was generically called values here.

Reply:
ok, fixed.